Transport.php

```diff
@@ -72,4 +80,73 @@ public static function exists($transport)
     {
         return in_array(strtolower($transport), self::get());
     }
+
+    /**
+     * Register a new wrapper in the collection
+     *
+     * @param  string   $protocol The registered protocol ws, wss, http...
```

Comment:
“The registered protocol (scheme://).” No example, placeholders or patterns instead.